webapp/channels/src/components/profile_popover/profile_popover_controller.tsx

```diff
@@ -126,6 +136,7 @@ export function ProfilePopoverController<TriggerComponentType = HTMLSpanElement>
                                 ref={refs.setFloating}
                                 style={{...floatingStyles, ...transitionStyles}}
                                 className={classNames('user-profile-popover', A11yClassNames.POPUP)}
+                                aria-label={props.username ? `${props.username}${userProfileAriaLabel}` : profileAriaLabel}
```

Comment:
This is still incorrect. Not all languages follow the syntax <subject><object", for example, this will fail for Urdu, Arabic, or any other right-to-left language. I'll illustrate with an example-

`Harshil's profile popover` translates to `Popover do perfil de Harshil` in Portuguese. No amount of localization of `profile_popover.aria_label.user` is going to make it look correct.

What we need to do is-

```
const profileAriaLabel = intl.formatMessage({
    id: 'profile_popover.aria_label',
    defaultMessage: '{userName}\'s profile popover',
    values: {
        userName: props.username,
    },
});
```

This way, `profile_popover.aria_label` can be correctly represented in all languages, whether they are LTR, or RTL or don't follow the English's grammar structure.